physionet-django/notification/templates/notification/email/notify_user_invited_managing_requests.html

```diff
@@ -0,0 +1,9 @@
+Dear {{ invitation.reviewer.get_full_name }},
+
+You have been invited to review access requests for the project {{ invitation.project }}.
+
+You can view and respond to requests following this link:
```

Comment:
perhaps add "with" or "via" (e.g. `You can view and respond to requests with the following link:`).